pages/api/request/help.ts

```diff
@@ -0,0 +1,88 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { cors, middleware, sessionExists } from "@lib/middleware";
+import { NotifyClient } from "notifications-node-client";
+import { logMessage } from "@lib/logger";
+import { MiddlewareProps } from "@lib/types";
+
+const SUPPORT_EMAIL_ADDRESS = "assistance+forms@cds-snc.freshdesk.com";
+const CONTACTUS_EMAIL_ADDRESS = "jose.jimenez@cds-snc.ca";
```

Comment:
Should these be environment variables. It doesn't seem like a good idea to have them hardcoded in the application

Reply:
@craigzour was there any take on this from @bryan-robitaille in the previous PR that contained and email https://github.com/cds-snc/platform-forms-client/pull/1242/files#diff-1bf840116a067de2cd60bd5404d945dd8e22a23bf079e14cde2801986b078e77R7

Reply:
@timarney I remember asking him if we should hide it under a Github secret. He said that it was fine to keep it in the code like that. cc @Moro-Code

Reply:
It's not about sensitivity as opposed to the path to modify. If this needs to be changed we would have to change lines of code as opposed to config. Just my opinion

Reply:
We could add this as a sort of Tech Debt question? Then later if we decide to, move any emails to env vars?

Reply:
@Moro-Code Oh yes I see what you mean. I guess the .env variable would now be more appropriate since there is two different places where we have the same email address set as a const in the code.

Whether it is done now or later (new ticket required) is another question and will depend on how important we need this code to be merged.

Reply:
Outside of .env vars does this project have a central spot for global type constants so at least it doesn't end up in multiple files and make it easy to update?

i.e. one place to change vs X # of files to search and change

Reply:
We do have a types folder. https://github.com/cds-snc/platform-forms-client/tree/develop/lib/types in the lib folder. We could add a constants file to have all the constants there so we could import them like so 

```TS
import { CONSTANT } from "@lib/types"
```